### PR TITLE
Docs: fix example in SparseCategoricalCrossEntropy

### PR DESCRIPTION
**Pull request by ddobbelaere**
Fixes the example in the docstring of `tf.keras.losses.SparseCategoricalCrossEntropy` to the one used in `tf.keras.losses.CategoricalCrossEntropy`.

Note that the current example is non-sensical as `[.5, .89, .6]` is not normalized (but `[.05, .89, .06]` is normalized, so it's probably just a typo).

### CODE REVIEW

**Comment by ddobbelaere** [comment]
@googlebot I signed it!

**Review by pavithrasv — APPROVED** [review]

**Comment by gbaned** [comment]
@ddobbelaere Can you please resolve conflicts? Thanks!

**Comment by ddobbelaere** [comment]
This PR has been superseded by another one that also fixes the docstring.
